.github/workflows/images.yaml

```diff
@@ -65,10 +73,10 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          file: ${{ matrix.dockerfile }}
+          file: ${{ matrix.image.dockerfile }}
           platforms: linux/amd64,linux/arm64
           push: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') }}
-          tags: |
-            ghcr.io/${{ github.repository }}${{ steps.vars.outputs.suffix }}:${{ steps.vars.outputs.tag }}
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
what's `steps.meta.outputs.tags`, maybe less `ghcr.io/....`?

Reply:
`steps.meta.outputs.tags` includes `ghcr.io/...`. The image repos is set by the `meta` step using the `with.images` setting as input.